packages/nextjs-template/components/DendronBreadCrumb.tsx

```diff
@@ -27,7 +28,14 @@ export function DendronBreadCrumb(props: DendronCommonProps) {
   return (
     <Breadcrumb style={{ margin: "16px 0" }}>
       {_.map(noteParents, (note) => {
-        return <Breadcrumb.Item key={note.id}>{note.title}</Breadcrumb.Item>;
+        const dest = note.id === props.noteIndex.id ? "/" : `/notes/${note.id}`;
```

Comment:
can we wrap this in a utility function? `eg. getNoteUrl({note, noteIndex)`

Reply:
done!